### PR TITLE
Add new command: `imgmod key show`

### PR DESCRIPTION
**Pull request by ccollins476ad**
```
show - Displays JSON describing one or more keys

Usage:
  imgmod key show <key-file> [key-files...] [flags]
```

### CODE REVIEW

**Comment by utzig** [comment]
Please rebase.

**Comment by ccollins476ad** [comment]
Thanks @utzig.  I have rebased.

File: ikey/ikey.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+func signKeyToDesc(key sec.PubSignKey, private bool) (Desc, error) {
+	var alg string
+	if key.Rsa != nil {
+		alg = fmt.Sprintf("RSA-%d", key.Rsa.Size()*8)
+	} else if key.Ec != nil {
+		alg = fmt.Sprintf("ECDSA-%d", key.Ec.X.BitLen())
+	} else {
+		alg = fmt.Sprintf("ED25519-%d", len(key.Ed25519)*8)
```
Comment (utzig) [inline ikey/ikey.go]: This text could as well read `ED25519` because the size is fixed!

**Review by utzig — APPROVED** [review]
LGTM

File: ikey/ikey.go (hunk 1 above)
Comment (ccollins476ad) [reply ikey/ikey.go]: Thanks @utzig, that is good to know.  I have fixed it as follows:
```
diff --git a/ikey/ikey.go b/ikey/ikey.go
index 54f0093..043d097 100644
--- a/ikey/ikey.go
+++ b/ikey/ikey.go
@@ -21,7 +21,7 @@ func signKeyToDesc(key sec.PubSignKey, private bool) (Desc, error) {
    } else if key.Ec != nil {
        alg = fmt.Sprintf("ECDSA-%d", key.Ec.X.BitLen())
    } else {
-       alg = fmt.Sprintf("ED25519-%d", len(key.Ed25519)*8)
+       alg = "ED25519"
    }

    pubBytes, err := key.Bytes()
```

**Pull request merged**